crispy_tailwind/templatetags/tailwind_field.py

```diff
@@ -76,41 +76,54 @@ def pairwise(iterable):
     return zip(a, a)
 
 
+@register.filter
+def tailwind_field_class(field):
```

Comment:
thought: how about a unit-test? this is a massive piece of python code, though not so much complicated logic.